src/components/Sidebar/components/SidebarRating/SidebarRating.tsx

```diff
@@ -0,0 +1,37 @@
+import { animationVariants } from "@/common/constants";
+import { Checkbox, Rating } from "@/components/UI";
+import commonStyles from "@/styles/Common.module.scss";
+import { motion } from "framer-motion";
+import { FC } from "react";
+import { v4 as uuidv4 } from "uuid";
+import styles from "./SidebarRating.module.scss";
+
+const START_VALUE = 5;
+const MIN_AMOUNT_STARS = 1;
+
+export const SidebarRating: FC = () => {
+  const renderRating = () => {
+    const ratingIcons = [];
+
+    for (let i = START_VALUE; i >= MIN_AMOUNT_STARS; i--) {
+      ratingIcons.push(
+        <li key={uuidv4()} className={styles.sidebarRatingItem}>
+          <Checkbox
+            id={uuidv4()}
+            name={<Rating amountRating={i} className={[styles.ratingList, styles.sidebarRatingIcon, styles.sidebarOutlineRatingIcon]} />}
+          />
+        </li>,
+      );
+    }
+    return ratingIcons;
+  };
+
```

Comment:
Would suggest to decompose rating stars list to a single UI component which accepts rating and generates the required stars amount.
This will enable the possibility to avoid writing logic for  generating stars list each time ( in a new place ) where it is used.